themes/bootstrap3/templates/Recommend/Databases.phtml

```diff
@@ -16,6 +17,16 @@
           </span>
         </div>
       <?php endforeach; ?>
+      <?php if ($linkToAll): ?>
+        <div class="list-group-item">
+          <span>
+            <a href="<?=$this->escapeHtmlAttr($this->proxyUrl($linkToAll))?>" class="title icon-link" target="_blank">
```

Comment:
I'm not sure whether using `proxyUrl` is valuable here; i.e. would any institution put their DatabasesAZ list behind a proxy?

Reply:
Actually @joecorrall may have an opinion on that

Reply:
Actually @joecorall may have an opinion on that

Reply:
> would any institution put their DatabasesAZ list behind a proxy?

They could if they didn't want to manage the proxy settings using Springshare's database asset management, and instead rely on EZProxy's auto-rewrite functionality to proxy links that need proxied in their various libguides pages.